pysrc/receive_message.py

```diff
@@ -1,6 +1,7 @@
 import base64
 import ast
 import random
+import json
```

Comment:
this is built in
prefer to using ast or whatever
because ast crashes when it encounters json style boolean literals